jax/numpy/fft.py

```diff
@@ -92,6 +92,36 @@ def ifftn(a, s=None, axes=None, norm=None):
   return _fft_core('ifftn', xla_client.FftType.IFFT, a, s, axes, norm)
 
 
+@_wraps(onp.fft.fft)
+def fft(a, n=None, axis=-1, norm=None):
+  if isinstance(axis,list) or isinstance(axis,tuple):
+    raise ValueError(
+      "jax.np.fft.fft does not support multiple axes. "
+      "Please use jax.np.fft.fftn. "
+      "Got axis %s." % (list(axis))
+    )
+
```

Comment:
This is probably fine, but maybe unnecessary, as long as a ValueError or TypeError gets raised instead of computing the wrong thing.

Reply:
Also as long as there's a clear error message! `testFftErrors` makes this easy to check, thanks for adding it.

Reply:
Sounds like a keeper then, thanks all.